src/components/filterbuilder/FilterBuilder.tsx

```diff
@@ -1,66 +1,78 @@
-import React, { useState, useCallback, useMemo } from "react";
+import React, { useState, useCallback, useEffect, useContext } from "react";
 import "./FilterBuilder.css";
 
 import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 import SaveIcon from "@material-ui/icons/Save";
+import Spinning from "../spinning";
+import VisibilityIcon from "@material-ui/icons/Visibility";
 
-import { FilterDefinition, SourceSystem } from "../../api";
-import { toMap, removeUndefined } from "../../utils";
+import { Filter, FilterDefinition, SourceSystem } from "../../api";
+import { removeUndefined } from "../../utils";
 
 import TagSelector, { Tag } from "../../components/tagselector";
 import SourceSelector from "../../components/sourceselector";
 
+import { FiltersContext, FiltersContextType } from "../../components/filters/contexts";
```

Comment:
The "../../components/filters/contexts" file is not included in this commit and does not exist on master.